src/Persistence_matrix/concept/PersistenceMatrixOptions.h

```diff
@@ -0,0 +1,181 @@
+/*    This file is part of the Gudhi Library - https://gudhi.inria.fr/ - which is released under MIT.
+ *    See file LICENSE or go to https://gudhi.inria.fr/licensing/ for full license details.
+ *    Author(s):       Hannah Schreiber
+ *
+ *    Copyright (C) 2024 Inria
+ *
+ *    Modification(s):
+ *      - YYYY/MM Author: Description of the modification
+ */
+
+/**
+ * @file PersistenceMatrixOptions.h
+ * @author Hannah Schreiber
+ * @brief Contains the concept for the matrix options.
+ */
+
+/// Gudhi namespace.
+namespace Gudhi {
+/// Persistence matrix namespace.
+namespace persistence_matrix {
+
+/** 
+ * @ingroup persistence_matrix
+ *
+ * @brief Concept of the template parameter for the class @ref Matrix.
+ *
+ * An implementation of this concept is @ref Default_options.
+ * If you want to provide your own, it is recommended that you derive from it and override some parts instead of
+ * writing a class from scratch.
+ */
+struct PersistenceMatrixOptions 
+{
+  /**
+   * @brief Field operators. Has to follow the @ref FieldOperators concept.
+   * The type will not be used if @ref is_z2 is set to true, so it can be set to anything.
+   */
+  using Field_coeff_operators = unspecified;
+  /**
+   * @brief Type for the dimension. Has to be an integer type.
+   * If unsigned, the maximal value of the type should not be attained during a run.
+   */
+  using dimension_type = unspecified;
+  /**
+   * @brief Type for the different indexation types and should be able to contain the maximal number of columns
+   * of the matrix during a run. Has to be an integer type.
+   * If unsigned, the maximal value of the type should not be attained during a run.
+   */
+  using index_type = unspecified;
+
+  /**
+   * @brief If true, indicates that the values contained in the matrix are in \f$ Z_2 \f$ and can therefore
+   * be treated like booleans. If set to false, the values are assumed to be in the field \f$ Z_p \f$ for
+   * some prime \f$ p \f$ given by @ref Field_coeff_operators. It is highly recommended to set the variable to true,
+   * if \f$ p = 2 \f$.
+   */
+  static const bool is_z2;
+  /**
+   * @brief Specifies the desired column type. All possible column types are described in @ref Column_types.
+   */
+  static const Column_types column_type;
```

Comment:
If a "matrix" actually contains 2 matrices (R and U), is it clear that they should use the same column type?